SimCalorimetry/HGCalSimProducers/python/hgcalDigitizer_cfi.py

```diff
@@ -240,7 +240,7 @@ def HGCal_setRealisticStartupNoise(process):
       (for instance turning on the 0th  bit turns off the impact of fluence in Si)
     """
     process=HGCal_setRealisticNoiseSi(process,byDose=True,byDoseAlgo=1)
-    process=HGCal_setRealisticNoiseSci(process,byDose=True,byDoseAlgo=2+8+16,referenceIdark=0.125,referenceXtalk=0.01)
+    process=HGCal_setRealisticNoiseSci(process,byDose=True,byDoseAlgo=2+16+32,byDoseFactor=0,referenceIdark=0.125,referenceXtalk=0.01)
```

Comment:
can you provide some explanation of the meaning of the numerical values in `byDoseAlgo`? (either in a comment, or by turning these into individually named variables rather than magic numbers)

Reply:
The meaning of the bits is explained in the header file: https://github.com/cms-sw/cmssw/blob/master/SimCalorimetry/HGCalSimAlgos/interface/HGCalSciNoiseMap.h#L14-L21
The bug was that previously 8 was suppressing the scaling of the signal light yield with the tile area - definitely not what we want. 32 is added at startup as it will force to ignore the the scaling of the noise with the fluence (and signal with dose).

Reply:
thanks @pfs !